src/HtmlUtils.tsx

```diff
@@ -346,7 +359,9 @@ function analyseEvent(content: IContent, highlights: Optional<string[]>, opts: E
             ? new HtmlHighlighter("mx_EventTile_searchHighlight", opts.highlightLink)
             : null;
 
-        if (isFormattedBody) {
+        if (isFormattedBody || opts.linkify) {
```

Comment:
The reason for this if-else to exist is that we want to do something particular (sanitize html + highlight) for events with html body and something else (just highlight) for events with just textual body. But we also want to linkify all events regardless of whether it has html body or not, so I would expect that to be done separately before this if-else. Otherwise, this code is going to be confusing to anyone who lacks the context of this PR.

Reply:
Hm.. fair point, I'll see what I can do about that.